mycroft/configuration/config.py

```diff
@@ -17,15 +17,15 @@
 import re
 import json
 import inflection
-from os.path import exists, isfile
+from os.path import exists, isfile, join, expanduser, dirname
 from requests import RequestException
+from xdg import BaseDirectory
 
+from .locations import DEFAULT_CONFIG, OLD_USER_CONFIG, USER_CONFIG
```

Comment:
why were these moved?  I think in general we put the relative imports after the `mycroft` package imports... or is that just me?

Reply:
The location of these has been restored